docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
I don't think scratch includes ca-certificates. You need ca-certificates to connect to https urls.